uuid.js

```diff
@@ -11,6 +11,9 @@
   // returns 128-bits of randomness, since that's what's usually required
   var _rng;
 
+  // Allow for MSIE11 msCrypto
+  var _crypto = _global.crypto || _global.msCrypto;
```

Comment:
Can you reverse the order here?   I see some reports that IE11 defines `crypto`, which would render the above moot.  I.e., instead do this:
  
    var _crypto = _global.msCrypto || _global.crypto;